.editorconfig

```diff
@@ -9,3 +9,6 @@ indent_style = space
 endofline = lf
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.json]
+indent_size = 2
```

Comment:
I don't understand why I rebased the branch to the master before I created the PR.